redash/query_runner/big_query.py

```diff
@@ -92,7 +94,17 @@ def enabled(cls):
 
     @classmethod
     def configuration_schema(cls):
-        return {
+        if EXPOSE_COST:
+            schema['order'].append('cost_per_tb')
+            schema['properties'].update({
+                'cost_per_tb': {
+                    'type': 'number',
+                    'title': 'Google Big Query cost per Tb scanned',
+                    'default': 1.1
```

Comment:
Default pricing for querying is $5 per TB, no?

Reply:
o.. yaa my bad it was BigQuery Storage API pricing 

$1.10 per TB read | The BigQuery Storage API is not included in the free tier.
-- | --

will be edit asap